### PR TITLE
file: fix CORS on File.prototype.content

### LINKED ISSUE #123: file.content invoke incorrect URL
`file.content()` API seems to invoke the [wrong URL](https://github.com/textileio/js-http-client/blob/4d9017be8c815c3acd60ebab8623b0068642a127/src/modules/file.ts#L18):
`const response = await this.sendGet(file/${hash}/data)`, which caused redirect to `file/xxx/content` (the correct URL), which then triggered the CORS preflight error.

### PR DESCRIPTION
**Pull request by balupton**
Fixes https://github.com/textileio/js-http-client/issues/123

/ref https://textile-public.slack.com/archives/CCRJZJT26/p1560800318046700

### CODE REVIEW

File: src/modules/file.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export default class File extends API {
    * @returns Raw data
    */
   async content(hash: string) {
-    const response = await this.sendGet(`file/${hash}/data`)
+    const response = await this.sendGet(`file/${hash}/content`)
```
Comment (carsonfarmer) [inline src/modules/file.ts:18]: Nice and easy. @balupton, this is going to come up with other redirects, which I like having available, as it means we can be a little more flexible with releasing updates to js-http-client. Any idea what the best way is to more permanently fix these CORS stoppers on the Go side of things?

**Review by carsonfarmer — APPROVED** [review]

**Comment by carsonfarmer** [comment]
I'm going to just get this one merged @balupton. I think if I don't squash, you'll get the credit...

File: src/modules/file.ts (hunk 1 above)
Comment (balupton) [reply src/modules/file.ts:18]: I'm not sure, as any redirect is a mark of deprecation, so we should coordinate immediate parch releases of this api client whenever we do such things with go-tectile. Patch releases will be adopted automatically by consumers of this client. This client is also the only one affected by cors.

**Comment by balupton** [comment]
> I'm going to just get this one merged @balupton. I think if I don't squash, you'll get the credit...

Squashing co-authors the commit, which is fine and dandy.

**Pull request merged**